### PR TITLE
use the same C() everywhere for absent / present options

### PR DESCRIPTION
**Pull request by zikalino**
##### SUMMARY
Just next doc cleanup PR....

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
multiple

### CODE REVIEW

File: lib/ansible/modules/cloud/azure/azure_rm_cdnendpoint.py [hunk 1]
```diff
@@ -33,16 +33,16 @@
...
     purge:
         description:
-            - Use with I(state) 'present' to purge the endpoint. Set to true to have the endpoint be purged.
+            - Use with I(state) C(present) to purge the endpoint. Set to true to have the endpoint be purged.
         type: bool
         default: false
     purge_content_paths:
         description:
-            - Use with I(state) 'present' and I(purge) 'true' to specify content paths to be purged.
+            - Use with I(state) C(present) and I(purge) 'true' to specify content paths to be purged.
```
Comment (acozine) [inline lib/ansible/modules/cloud/azure/azure_rm_cdnendpoint.py]:
```suggestion
            - Use with I(state) C(present) and I(purge) C(true) to specify content paths to be purged.
```

File: lib/ansible/modules/cloud/azure/azure_rm_cdnendpoint.py [hunk 2]
```diff
@@ -33,16 +33,16 @@
             - Valid azure location. Defaults to location of the resource group.
     started:
         description:
-            - Use with I(state) 'present' to start the endpoint. Set to false to stop the endpoint. Set to true to start the endpoint.
+            - Use with I(state) C(present) to start the endpoint. Set to false to stop the endpoint. Set to true to start the endpoint.
```
Comment (acozine) [inline lib/ansible/modules/cloud/azure/azure_rm_cdnendpoint.py]:
```suggestion
            - Use with I(state) C(present) to start the endpoint. Set to C(false) to stop the endpoint. Set to C(true) to start the endpoint.
```

File: lib/ansible/modules/cloud/azure/azure_rm_cdnendpoint.py [hunk 3]
```diff
@@ -33,16 +33,16 @@
...
         type: bool
     purge:
         description:
-            - Use with I(state) 'present' to purge the endpoint. Set to true to have the endpoint be purged.
+            - Use with I(state) C(present) to purge the endpoint. Set to true to have the endpoint be purged.
```
Comment (acozine) [inline lib/ansible/modules/cloud/azure/azure_rm_cdnendpoint.py]:
```suggestion
            - Use with I(state) C(present) to purge the endpoint. Set to C(true) to have the endpoint be purged.
```

File: lib/ansible/modules/cloud/azure/azure_rm_resourcegroup.py [hunk 4]
```diff
@@ -24,7 +24,7 @@
 options:
     force_delete_nonempty:
         description:
-            - Remove a resource group and all associated resources. Use with state 'absent' to delete a resource
+            - Remove a resource group and all associated resources. Use with state C(absent) to delete a resource
```
Comment (acozine) [inline lib/ansible/modules/cloud/azure/azure_rm_resourcegroup.py]:
```suggestion
            - Remove a resource group and all associated resources. Use with state C(absent) to delete a resource.
```

**Review by acozine — COMMENTED** [review]
Looks good - I added a couple of suggestions.

**Comment by zikalino** [comment]
@acozine thanks! commited suggestions

**Comment by acozine** [comment]
Thanks @zikalino for making the azure docs more consistent and more readable.

**Pull request merged**